src/Symfony/Component/Routing/Annotation/Route.php

```diff
@@ -54,7 +54,7 @@ public function __construct(
         array $options = [],
         array $defaults = [],
         string $host = null,
-        array $methods = [],
+        $methods = [],
         array $schemes = [],
```

Comment:
can you also fix it for `schemes`, for which the setter also support `string|string[]` ?

Reply:
Good catch, it's done.